pkg/watch/mux.go

```diff
@@ -162,6 +165,7 @@ func (m *Broadcaster) loop() {
 		}
 		m.distribute(event)
 	}
+	m.distributing.Done()
```

Comment:
Distribute is called synchronously above. Once we get to this point, no more writes to the outgoing channels will occur, unless you count the close call, which I do. After closeAll is called, then there really will not be any more writes to the channels and it should be safe to tell the waitgroup that we're done.